components/prism-hoon.js

```diff
@@ -0,0 +1,21 @@
+Prism.languages.hoon = {
+	'constant': /%(?:\.[ny]|[\w-]+)/,
+	'comment': {
+		pattern: /::.*/,
+		lookahead: true,
+		greedy: true
+	},
+	'function': /(?:\+[-+] {2})?(?:[a-z]([a-z0-9-]*[a-z0-9])?)/,
```

Comment:
You missed a capturing group :)

```suggestion
	'function': /(?:\+[-+] {2})?(?:[a-z](?:[a-z0-9-]*[a-z0-9])?)/,
```